src/types/positions.ts

```diff
@@ -91,6 +91,7 @@ export interface EarnDataProps {
   earningItems: EarningItem[]
   depositTokenId: string
   withdrawTokenId: string
+  rewardPositionIds?: string[]
```

Comment:
nit: we've been using the plural `rewards` more often. so I suggest `rewardsPositionIds`